ox-reveal.el

```diff
@@ -1406,17 +1406,17 @@ transformed fragment attribute to ELEM's attr_html plist."
                                    frag-list))
                       (items (org-element-contents elem))
 		      (default-style-list
-                            (mapcar (lambda (a) default-style)
-                                    (number-sequence 1 (length items)))))
+                        (mapcar (lambda (a) default-style)
+                                (number-sequence 1 (length items)))))
                  (if frag-index
-                     (mapcar* 'org-reveal--update-attr-html
-                              items frag-list default-style-list (car (read-from-string frag-index)))
+                     (cl-mapcar 'org-reveal--update-attr-html
```

Comment:
Same question as cl-case

Reply:
Same reasoning as `cl-case`: `mapcar*` is deprecated along with cl.el.